extra/release_tool.py

```diff
@@ -237,18 +234,34 @@ def get_value_from_password_storage(server, key):
         if "/" in server:
             server = server[:server.index("/")]
 
-        output = subprocess.check_output(["pass", "find", server]).decode()
-        count = 0
-        server_path = []
-        for line in output.split('\n'):
-            if line.lower().startswith("search terms: ") or line == "":
-                continue
-            server_path.append(re.match(pass_tree_re, line).group(1))
-            count += 1
-        if count == 0:
-            return
+        pass_dir = os.getenv("PASSWORD_STORE_DIR")
+        if not pass_dir:
+            pass_dir = os.path.join(os.getenv("HOME"), ".password-store")
+
+        server_path_str = os.getenv("PASS_GITLAB_COM")
+
+        if not server_path_str:
+            output = subprocess.check_output(["find", pass_dir, "-type", "f", "-path", "*%s*" % server]).decode()
```

Comment:
Aha! So using `find -path` is what I was missing when I did the first implementation. I couldn't figure out how to parse the tree like output of pass...

Thanks!

Reply:
Hehe yes, this was after several iterations of frustratingly trying to parse `pass` output. It really has a very difficult to parse format, `find` is much easier.